### DISCUSSION TITLE
电子新人关于各种板子的疑问

### DISCUSSION THREAD

**Discussion by ma9icloud** [peng-zhihui/ElectronBot#235] [Q&A]
由于没有接触过电子电路所以提的问题比较基础, 期待有人解答

现在的项目主要由4块板子构成: ElectronBot, SensorBoard, BaseConnector, ServoDrive(-DK)
1. 可不可以只使用一块主控板连接上所有的元器件, 降低内部占用空间
2. 可不可以进一步简化, 所有的运算都在上位机完成, 内部电路板只负责:
    - 把上位机的命令输出成舵机的pwm, 显示屏输出和音频输出
    - 把传感器信号传给上位机

以上简化会有什么问题吗?(增加控制延迟?)

**Comment by Kitem** [comment]
您好，我是焕挺，您的邮件已经收到，我会尽快回复！
